### PR TITLE
[IT] Week07 lect 02 and 03 (+ small cleaning on other files)

### PR DESCRIPTION
**Pull request by marcozullich**
I also made some corrections on EN 07-2 and 07-3. Main corrections are typos, double blanks, missing blanks, missing mathematical env $ ... $

### CODE REVIEW

File: docs/it/week03/03.md [hunk 1]
```diff
@@ -23,6 +23,6 @@ Diamo un'introduzione su come le CNN si sono evolute nel tempo. Discutiamo in de
 
 ## Pratica
 
-Sono discusse nel dettaglio quelle proprietà dei segnali naurali che sono più pertinenti per le CNN, ovvero: localizzazione, stazionarietà e composizionalità. Esploriamo precisamente come un filtro utilizza queste proprietà attraverso la sparsità, la condivisione dei pesi e le pile di strati, motivando anche i concetti di *padding* e *pooling*. Infine, paragoniamo le performance fra FCN e CNN per diverse *data modality*.
+Sono discusse nel dettaglio quelle proprietà dei segnali naurali che sono più pertinenti per le CNN, ovvero: localizzazione, stazionarietà e composizionalità. Esploriamo precisamente come un kernel utilizza queste proprietà attraverso la sparsità, la condivisione dei pesi e le pile di strati, motivando anche i concetti di *padding* e *pooling*. Infine, paragoniamo le performance fra FCN e CNN per diverse *data modality*.
```
Comment (Atcold) [inline docs/it/week03/03.md:26]:
```suggestion
Sono discusse nel dettaglio quelle proprietà dei segnali naurali che sono più pertinenti per le CNN, ovvero: localizzazione, stazionarietà e composizionalità. Esploriamo precisamente come un filtro utilizza queste proprietà attraverso la sparsità, la condivisione dei pesi e le pile di strati, motivando anche i concetti di *padding* e *pooling*. Infine, paragoniamo le performance fra FCN e CNN per diverse *data modality*.
```

File: docs/it/week03/03.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ translator: marcozullich
 
 ## Lezione parte A
 
-Prima di tutto, diamo un'occhiata alla visualizzzione di una rete neurale composta da 6 strati. Dopodiché, discutiamo di convoluzioni e reti neurali convoluzionali (CNN). Rivediamo numerose varianti di trasformazione dei parametri nel contesto delle CNN e introduciamo l'idea di filtro (_kernel_), che viene utilizzato per imparare caratteristiche in maniera gerarchica, permettendoci dunque di classificare i nostri dati di input, il che è l'idea di fondo che motiva l'utilizzo delle CNN.
+Prima di tutto, diamo un'occhiata alla visualizzzione di una rete neurale composta da 6 strati. Dopodiché, discutiamo di convoluzioni e reti neurali convoluzionali (CNN). Rivediamo numerose varianti di trasformazione dei parametri nel contesto delle CNN e introduciamo l'idea di kernel, che viene utilizzato per imparare caratteristiche in maniera gerarchica, permettendoci dunque di classificare i nostri dati di input, il che è l'idea di fondo che motiva l'utilizzo delle CNN.
```
Comment (Atcold) [inline docs/it/week03/03.md:12]:
```suggestion
Prima di tutto, diamo un'occhiata alla visualizzzione di una rete neurale composta da 6 strati. Dopodiché, discutiamo di convoluzioni e reti neurali convoluzionali (CNN). Rivediamo numerose varianti di trasformazione dei parametri nel contesto delle CNN e introduciamo l'idea di filtro (_kernel_), che viene utilizzato per imparare caratteristiche in maniera gerarchica, permettendoci dunque di classificare i nostri dati di input, il che è l'idea di fondo che motiva l'utilizzo delle CNN.
```

**Comment by Atcold** [comment]
Thank you.
Next time, can you submit two different PRs? One for the translation and one for the corrections?
Thanks!

**Comment by Atcold** [comment]
Lecture 7-3 will be expanded (there was a part missing).

**Comment by Atcold** [comment]
Here there's is an update about `07-3` https://github.com/Atcold/pytorch-Deep-Learning/pull/327
When you have time, please update the Italian translation.

**Comment by marcozullich** [comment]
Ok, I'll start working on it this evening CET.

**Pull request merged**